File: JS/assign1p2.js
/**
  * @brief Assignment 1 Problem 2
  * @author Daidemang Francis
  */

var x1 = 5; 
var x2 = 6; 
var x3 = 7; 
var perimeter = (x1 + x2 + x3) / 2;
var area =  Math.sqrt(perimeter*((perimeter-x1)*(perimeter-x2)*(perimeter-x3)));
console.log(area);

File: c++/Fall 2018/WindChill/assign1p2.cpp
/**
  * @brief Assignment 1 Problem 2
  * @author Francis
  */
#include <iostream>
#include <math.h>
using namespace std;

int main()
{
    double temperature, windChill;
    int windSpeed;

    // get user input
    cout << "Enter the temperature in Fahrenheit: ";
    cin >> temperature;
    cout << "Enter the windchill in MPH: ";
    cin >> windSpeed;

    // compute windchill
    windChill = 35.74 + (0.6125 * temperature) - 35.75 * pow(windSpeed, 0.16) + (0.4275 * temperature) * pow(windSpeed, 0.16);
    cout << "The wind chill index is " << windChill << endl;

    return 0;
}
